.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+go:
+    - 1.7
```

Comment:
Swapped in 1.6 for tip, sure.

Before it wouldn't have worked because I accidentally imported `context` (go1.7 stdlib only), so I made that minor change as well.